django/core/checks/model_checks.py

```diff
@@ -25,6 +29,20 @@ def check_all_models(app_configs=None, **kwargs):
             )
         else:
             errors.extend(model.check(**kwargs))
+    duplicate_db_table_list = [
+        db_table_attr for db_table_attr, model_list in db_table_attr_map.items()
+        if len(model_list) > 1
+    ]
+    if duplicate_db_table_list:
+        for db_table in duplicate_db_table_list:
+            errors.append(
+                Error(
+                    "db_table '%s' was declared in multiple models: '%s'"
+                    % (db_table, ', '.join(db_table_attr_map[db_table])),
+                    obj=model,
```

Comment:
I feel like `obj` should be `db_table` here, `model` is a reference to the last value of the `for model in models` loop above and is completely unrelated.

Reply:
I think it should ideally be the label of one of the model classes that caused the error instead of the `db_table` attribute of the model's meta.  But I'm not pretty sure about it.

I will change it to db_table and update the PR and then we can see what would be better.